scripts/feature_tests/upgrade/common.sh

```diff
@@ -0,0 +1,432 @@
+#!/bin/bash
+
+export IMAGE_OS=${IMAGE_OS:-"Ubuntu"}
+export IMAGE_USERNAME=${IMAGE_USERNAME:-"metal3"}
+export KUBERNETES_VERSION="v1.18.0"
+export KUBERNETES_BINARIES_VERSION="v1.18.0"
+export UPGRADED_K8S_VERSION_1="v1.18.1"
+export UPGRADED_K8S_VERSION_2="v1.18.2"
+export UPGRADED_BINARY_VERSION="v1.18.1"
+export CLUSTER_APIENDPOINT_IP=${CLUSTER_APIENDPOINT_IP:-"192.168.111.249"}
+# The next lines to be fixed in the CI
+export CAPM3PATH="/home/ubuntu/go/src/github.com/metal3-io/cluster-api-provider-metal3"
+
+function generate_metal3MachineTemplate() {
+    NAME="${1}"
+    CLUSTER_UID="${2}"
+    Metal3MachineTemplate_OUTPUT_FILE="${3}"
+
+    echo "
+apiVersion: infrastructure.cluster.x-k8s.io/v1alpha3
+kind: Metal3MachineTemplate
+metadata:
+  name: ${NAME}
+  namespace: metal3
+  ownerReferences:
+  - apiVersion: cluster.x-k8s.io/v1alpha3
+    kind: Cluster
+    name: test1
+    uid: ${CLUSTER_UID}
+spec:
+  template:
+    spec:
+      hostSelector: {}
+      image:
+        checksum: http://172.22.0.1/images/bionic-server-cloudimg-amd64-raw.img.md5sum
+        checksumType: md5
+        format: raw
+        url: http://172.22.0.1/images/bionic-server-cloudimg-amd64-raw.img
```

Comment:
checksum and url are parameterized in #357

Reply:
common.sh renamed to upgrade_common.sh, this PR updated accordingly